common/src/main/java/org/apache/gravitino/dto/responses/RoleResponse.java

```diff
@@ -64,8 +64,5 @@ public void validate() throws IllegalArgumentException {
     Preconditions.checkArgument(
         StringUtils.isNotBlank(role.name()), "role 'name' must not be null and empty");
     Preconditions.checkArgument(role.auditInfo() != null, "role 'auditInfo' must not be null");
-    Preconditions.checkArgument(
-        role.securableObjects() != null && !role.securableObjects().isEmpty(),
-        "role 'securableObjects' can't null or empty");
```

Comment:
Is it still removable, or should be checked?

Reply:
Ditto. Modified, too.